### PR TITLE
draft for CLI filename completion

### PR DESCRIPTION
**Pull request by jomiklos**

### CODE REVIEW

File: python/drned_xmnr/action.py [hunk 1]
```diff
@@ -83,6 +84,39 @@ def action_response(self, uinfo, result, output):
...
+        try:
+            # TODO - escape/sanitize input "token"
+            pattern_to_match = str(token)
+            if '*' not in pattern_to_match:
+                pattern_to_match += '*'
+            self.log.debug('Find completions for: \"' + pattern_to_match + '\"')
+            matched_paths = glob.glob(pattern_to_match)
+            tv = []
+            for path in matched_paths:
+                tv.append((dp.COMPLETION, str(path), None))
```
Comment (martin-volf) [inline python/drned_xmnr/action.py]: use list comprehension, please

File: python/drned_xmnr/action.py [hunk 2]
```diff
@@ -83,6 +84,39 @@ def action_response(self, uinfo, result, output):
...
+            self.log.debug('Found matches:' + str(tv))
```
Comment (martin-volf) [inline python/drned_xmnr/action.py]: I'd be more careful with logging - the completion might be invoked several times for one CLI line, no need to debug-log really everything

File: python/drned_xmnr/action.py (hunk 1 above)
Comment (jomiklos) [reply python/drned_xmnr/action.py]: fixed in succeeding commit...

File: python/drned_xmnr/action.py (hunk 2 above)
Comment (jomiklos) [reply python/drned_xmnr/action.py]: fixed in succeeding commit...

**Pull request merged**